submissions/Ant-C-tech/tiny-js-world/index.js

```diff
@@ -0,0 +1,97 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Ant-C-tech/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Spike',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof-Woof!',
+   friends: ['John'],
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Tom',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Meow-Meow!',
+   friends: ['John'],
+};
+
+const woman = {
+   species: 'human',
+   name: 'Jane',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Nice to meet you!',
+};
+
+const man = {
+   friends: ['Spike', 'Tom'],
+   species: 'human',
+   name: 'John',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi, how are you?',
+   gender: 'male',
+};
+
+function createGreetingContent(obj) {
+
+   const propPrefixes = {
+      species: 'I am a %. ',
+      name: 'My name is %. ',
+      gender: 'My gender is %. ',
+      hands: 'I have % hands. ',
+      legs: 'I have % legs. ',
+      saying: 'I want to say you "%". ',
+      friends: 'I have friend %. ',
+   };
+
+  const greeting = Object.keys(propPrefixes).map((key) => {
+      if (Array.isArray(obj[key])) {
+         for (const item of obj[key]) {
+            return propPrefixes[key].replace('%', item);
+         };
+      } else if (obj[key] && propPrefixes[key]) {
+         return propPrefixes[key].replace('%', obj[key]);
+      };
+   });
+
+   return greeting.join('');
```

Comment:
There is a pattern in your props templates that can be generalized as a more appropriate parameter for `join`.

Reply:
Thank you for your review! I have tried to resolve this issue.